main.py

```diff
@@ -2,6 +2,37 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        ("Western",),
+        ("Action",),
+        ("Dramma",)
+    ]
```

Comment:
There's no need for genres to be a list of tuples if you're only using a single field (name) for the Genre model. it can be a simple list of names